manga_translator/manga_translator.py

```
@@ -3528,25 +3593,24 @@ async def translate_batch(self, images_with_configs: List[tuple], batch_size: in
                 is_hq_translator = translator_type in [Translator.openai_hq, Translator.gemini_hq]
                 is_import_export_mode = self.load_text or self.template or self.translate_json_only
 
-                # 如果是高质量翻译且未启用并发模式，使用专用的高质量翻译流程
-                if is_hq_translator and not is_import_export_mode and not self.batch_concurrent:
+                # 如果是高质量翻译且未启用有效并发模式，使用专用的高质量翻译流程
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)